aspnetcore/tutorials/publish-to-azure-webapp-using-vscode.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Publish an ASP.NET Core app to Azure with Visual Studio Code
+author: ricardoserradas
+description: Learn how to publish an ASP.NET Core app to Azure App Service using Visual Studio Code
+ms.author: ricardoserradas
+ms.custom: mvc
+ms.date: 04/16/2019
+uid: tutorials/publish-to-azure-webapp-using-vscode
+---
+
+# Publish an ASP.NET Core app to Azure with Visual Studio Code
+
+By [Ricardo Serradas](https://twitter.com/ricardoserradas)
+
+[!INCLUDE [Azure App Service Preview Notice](../includes/azure-apps-preview-notice.md)]
+
+To troubleshoot an App Service deployment issue, see <xref:host-and-deploy/azure-apps/troubleshoot>.
+
+## Intro
+
+With this tutorial, you'll learn how to create an ASP.Net Core MVC Application
+and deploy it within Visual Studio Code.
+
+## Set up
+
+- Open a [free Azure account](https://azure.microsoft.com/free/dotnet/) if you don't have one.
+- Install [.NET Core](https://dotnet.microsoft.com/download)
+- Install [Visual Studio Code](https://code.visualstudio.com/Download)
+  - Instal the [Azure App Service Extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice) to Visual Studio Code and configure it before proceeding
```

Comment:
You also need to install the C# extension to build and compile .NET Core projects. Can you please add guidance to do so before installing the "Azure App Service Extension".
Something along the lines of:
- Install the [C# Extension] (https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp) to Visual Studio Code.

Reply:
Done. Your suggestion was used here, thanks!